src/RMGOutputTools.cc

```diff
@@ -110,11 +110,53 @@ RMGDetectorHit* RMGOutputTools::average_hits(std::vector<RMGDetectorHit*> hits,
   return hit;
 }
 
+void RMGOutputTools::redistribute_gamma_energy(std::map<int, std::vector<RMGDetectorHit*>> hits_map,
+    RMGOutputTools::ClusterPars cluster_pars, bool has_distance_to_surface) {
+
+  RMGLog::Out(RMGLog::debug, "Merging gamma tracks ");
+
+
+  // for tracks of gammas look for a step close to each post-step point
+  // to redistribute the energy to
+  for (const auto& [trackid, input_hits] : hits_map) {
+
+    // only apply to gamma
+    if (input_hits.front()->particle_type != 22) continue;
```

Comment:
I don't like to hardcode the particle id (22). I think there is a better way by using `G4Gamma`

Reply:
like maybe https://geant4.kek.jp/Reference/11.3.0/classG4ParticleDefinition.html#a1d8cd4bb79a1161d62ec97f91ccf4a5a

Reply:
also above in line 34, and also with 11 for electrons below